Source/setup/02-deploy-apis.sh

```diff
@@ -53,7 +53,8 @@ while [ "$1" != "" ]; do
         -d | --dns)                     shift
                                         dns=$1
                                         ;;
-        --httpRouting)                  httpRouting=1
+        --httpRouting)                  shift
```

Comment:
I think that the "shift" in --httRouting is **not correct**: switch parameters can't have a "shift". If they do, next parameter name will be swallowed resulting in parameter errors.
Please @ivilches can review and fix that?

Reply:
You're right! Fixed with the commit below.